.github/workflows/test_python.yml

```diff
@@ -62,6 +62,10 @@ jobs:
       run: |
         sed "s/# android.accept_sdk_license = False/android.accept_sdk_license = True/" -i buildozer.spec
         buildozer android p4a -- --help
+    - name: buildozer android debug
+      run: |
+        touch main.py
+        buildozer android debug
```

Comment:
Adds a 11 minutes overhead, but it's probably worth trying until we have proper unit test coverage :cry:

Reply:
Oh boy, this is quite of time, but you are right, better this than nothing :+1: